content/sensu-go/5.16/api/entities.md

```diff
@@ -213,7 +244,10 @@ The `/entities/:entity` API endpoint provides HTTP GET access to [entity data][1
 In the following example, querying the `/entities/:entity` API endpoint returns a JSON map that contains the requested [`:entity` definition][1] (in this example, for the `:entity` named `sensu-centos`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/entities/sensu-centos -H "Authorization: Bearer $SENSU_TOKEN"
+curl X GET \
```

Comment:
Should the X have a dash in front of it?

Reply:
yes again 😬